src/main/java/seedu/exercise/logic/parser/SuggestCommandParser.java

```diff
@@ -32,9 +41,23 @@ public SuggestCommand parse(String args) throws ParseException {
             return new SuggestBasicCommand();
         }
 
+        if (suggestType.equals("possible")) {
+            return parsePossible(argMultimap);
+        }
+
         throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, SuggestCommand.MESSAGE_USAGE));
     }
 
+    private Prefix[] getPrefixes() {
```

Comment:
Why the choice of an array here instead of just returning a list?

Reply:
The tokenize method in ArgumentTokenizer takes in prefixes in the form of Varargs. So i had to use an array instead of a list to represent the prefixes.

Reply:
noted